src/maga/highscore/Score.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package maga.highscore;
+
+/**
+ *
+ * @author ander
+ */
+public class Score {
+    
+    /** 
+     * The name for the player. 
+     */
+    private String name; 
+    
+    /**
+     * The int score for our HighScore.
+     */
+    private int score; 
+    
+    /**
+     * No args construtor. 
+     */
+    public Score() {   
+    }
```

Comment:
This should be removed

Reply:
fixed in: 17bcfa980f5403b84beb5453e18db902ada95885